app/js/views/element_phone_field_view.js

```diff
@@ -0,0 +1,85 @@
+import { View } from 'components/fxos-mvc/dist/mvc';
+import 'components/gaia-button/gaia-button';
+
+(function(window) {
+'use strict';
+
+var shadowTemplate =
+`<style scoped>
+#container {
```

Comment:
We should probably ask @wilsonpage but AFAIK, we've done it this way in several web components to get past some gaps in the platform regarding the styling of shadow DOM.